### DISCUSSION TITLE
Dexcom G6 vs xdrip+

### DISCUSSION THREAD

**Discussion by MatlM11** [NightscoutFoundation/xDrip#3679] [General]
Please, might I ask for an advice : Is possible to set up data transfer from dexcom  app g6 on one android phone to another phone with xdrip+? The functionality of xdrip as dexcom follow? Please give help how to set up.  BR Martin

**Comment by Navid200** [comment]
You need to set up the Dexcom app to share.  You need to invite a follower from Dexcom app.
Then, you need to install the official Dexcom share follow app on the other phone and make sure it works.

After that, on the follower phone, you can install xDrip and set its hardware data source to Dex share follower.
https://navid200.github.io/xDrip/docs/HardwareDataSource.html
You need to enter the user ID and password that the uploader uses under the Dex follow settings.  The US server needs to be a valid selection depending on where the master is located.

**Reply by MatlM11** [reply]
Dear, GREAT!!!, thank you very much Martin
